src/components/Button/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropType from 'prop-types';
+
+/* eslint-disable */
+function Button({ classList, buttonType, text, handleData }) {
+  /* eslint-enable */
```

Comment:
- [ ] Please remove the `eslint-disable` statements as this defies the purpose of using linters in the first place.